torchaudio/models/conformer.py

```diff
@@ -31,6 +31,7 @@ def __init__(
         num_channels: int,
         depthwise_kernel_size: int,
         bias: bool = False,
+        use_group_norm: bool = False,
```

Comment:
and similarly here?

also noticed that `dropout` is missing a docstring...we can address separately